src/riak_kv_bucket.erl

```diff
@@ -88,7 +111,9 @@
 %%
 %% There is no way to _remove_ a property
```

Comment:
Just have to say that I documented that really well following a sweary phone call, eh?